mm2src/coins/tendermint/tendermint_token.rs

```diff
@@ -60,9 +60,10 @@ impl Deref for TendermintToken {
     fn deref(&self) -> &Self::Target { &self.0 }
 }
 
-#[derive(Clone, Debug, Deserialize, Serialize)]
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
 pub struct TendermintTokenProtocolInfo {
     pub platform: String,
+    // Todo: can decimals be gotten from rpc call for custom tokens?
```

Comment:
There is [denoms_metadata](https://nucleus-api.bravo.komodo.earth/cosmos/bank/v1beta1/denoms_metadata) endpoint available on tendermint, but I am not sure if it works with the custom tokens.

Reply:
Thanks for the hint, will give it a look.